### ISSUE TITLE
Update crates.io?

### ISSUE THREAD

**Issue by jsgf** [mackwic/colored#61]
Hi - I was about to fix the conditional dependency on winconsole, but I see it's already fixed. Could you update the crates.io version with at least that fix?

**Comment by scottmcm** [comment]
@mackwic It would also be nice to get a new version out that includes #67 because that fixes the error that just showed up in crater: 
```rust
[INFO] [stderr] error[E0308]: mismatched types
[INFO] [stderr]   --> src/style.rs:76:23
[INFO] [stderr]    |
[INFO] [stderr] 76 |             .filter(|&&(ref mask, _)| (0 != (u & mask)))
[INFO] [stderr]    |                       ^^^^^^^^^^^^^^ expected tuple, found reference
[INFO] [stderr]    |
[INFO] [stderr]    = note: expected type `(u8, style::Styles)`
[INFO] [stderr]               found type `&_`
[INFO] [stderr] 
[INFO] [stderr] error[E0308]: mismatched types
[INFO] [stderr]   --> src/style.rs:77:19
[INFO] [stderr]    |
[INFO] [stderr] 77 |             .map(|&(_, value)| value)
[INFO] [stderr]    |                   ^^^^^^^^^^^ expected tuple, found reference
[INFO] [stderr]    |
[INFO] [stderr]    = note: expected type `(u8, style::Styles)`
[INFO] [stderr]               found type `&_`
```
https://crater-reports.s3.amazonaws.com/pr-65819/try%23b34fa46cc6fa5d46c6adcb041ebd28c7f7a374f3/reg/colored-1.8.0/log.txt

**Comment by mati865** [comment]
Published as version 1.9.

**Comment by jsgf** [comment]
Thanks
